docs/querying/sql-query-context.md

```diff
@@ -29,7 +29,7 @@ sidebar_label: "SQL query context"
 :::
 
 Druid supports query context parameters which affect [SQL query](./sql.md) planning.
```

Comment:
Suggest putting together a more coherent introduction:

Ex. paragraph 1
* what are they
* (what does that mean)
* link to how to set it
* link to reference page

Ex. paragraph 2: "you can override"

Ex. paragraph 3: "the following table lists"